tools/c7n_azure/c7n_azure/function_package.py

```diff
@@ -223,7 +226,11 @@ def publish(self, deployment_creds):
 
         self.log.info("Publishing Function package from %s" % self.pkg.path)
 
-        zip_file = open(self.pkg.path, 'rb').read()
+        # Windows requires TEMPORARY flag if you want to open files created by tempfile library
```

Comment:
aha, was wondering about this for a while, i had just assumed it wasn't supported, i would have expected the stdlib to have done that by default.